prisma/migrations/20230319022555_make_role_field_required_and_change_type_to_enum_in_access_table/migration.sql

```diff
@@ -0,0 +1,17 @@
+/*
+  Warnings:
+
+  - The `role` column on the `Access` table would be dropped and recreated. This will lead to data loss if there is data in the column.
+  - The `role` column on the `ProjectInvite` table would be dropped and recreated. This will lead to data loss if there is data in the column.
+
+*/
+-- CreateEnum
+CREATE TYPE "UserRole" AS ENUM ('maintainer', 'owner', 'editor', 'viewer', 'developer', 'guest');
```

Comment:
Let's limit roles to `'maintainer', 'owner', 'developer', 'guest'` for now. Down the line we will have scope based custom roles. It becomes harder to support multiple roles without scopes.